client-device-connection-infinispan/pom.xml

```diff
@@ -64,6 +64,11 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>redis.clients</groupId>
+      <artifactId>jedis</artifactId>
+      <version>4.3.1</version>
+    </dependency>
```

Comment:
Ignore this.

The whole module has been "disconnected" and refactored into 4 new modules. The `client-device-connection-infinispan` module will eventually be removed completely.